ntoskrnl/ke/i386/thrdini.c

```diff
@@ -505,6 +509,9 @@ KiDispatchInterrupt(VOID)
         NewThread->State = Running;
         OldThread->WaitReason = WrDispatchInt;
 
+        /* Lock the PRCB */
+        KiAcquirePrcbLock(Prcb);
```

Comment:
Maybe you can mention where it's going to be unlocked?

Reply:
Removed! It seems I fixed it another way in the SMP branch.